less/forum/Composer.less

```diff
@@ -114,7 +114,7 @@
     background: @body-bg;
 
     &:not(.minimized) {
-      position: absolute;
```

Comment:
My understanding is that this is intentional: we want people to be able to scroll up and down in a discussion while working on their reply. That being said, my understanding might be incorrect.

Reply:
oh, if that's the case then we wouldn't want to change this then. From the commit I mentioned above https://github.com/flarum/core/commit/e6e2cdd3e98be6268f4dd7fb370d366f565ae72e my understanding is that it was only done to fix a certain bug.

Reply:
Also, users can always minimize the composer to take a look at the discussion, so I'm not sure if scrolling up and down from there is really needed

Reply:
Not inherently opposed to changing, but let's revisit this in the next release cycle and hopefully have some discussion on it